qiskit_experiments/framework/base_experiment.py

```diff
@@ -39,14 +39,21 @@ def __init__(
         analysis: Optional[BaseAnalysis] = None,
         backend: Optional[Backend] = None,
         experiment_type: Optional[str] = None,
+        **experiment_options,
     ):
         """Initialize the experiment object.
 
+        `experiment_options` catches all keyword arguments not explicitly defined in `__init__`. As this
```

Comment:
This sentence should be removed entirely. It doesn't make sense muich sense for base class docs, and the Arg description should be enough

Reply:
Addressed in bbe6821.